scripts/ts-copy.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+// @flow
+
+const fs = require('fs-extra');
+const globby = require('globby');
+const path = require('path');
+
+(async () => {
+  const files = await globby(['src/**/*.d.ts']);
+  files.forEach(async file => {
+    const from = path.join(__dirname, '../', file);
+    const to = path.join(__dirname, '../', file.replace('src', 'dist'));
+    await fs.copy(from, to);
```

Comment:
node now has a copy method, so yo don't need this dep : https://nodejs.org/api/fs.html#fs_fs_copyfilesync_src_dest_flags

Reply:
Do we use Node v12 in our CI?

Reply:
Oh, it's already in 10